### ISSUE TITLE
Update generating testnodes for V18

### ISSUE THREAD

**Issue by cwaldron97** [bitcoin-s/bitcoin-s#552]
Currently `testkit/src/main/scala/org/bitcoins/testkit/rpc/BitcoindRpcTestUtil.scala` has trouble creating a testnode for v18 as `generate` has deprecated in v18.

**Comment by torkelrogstad** [comment]
So here's a suggestion for how to fix this: 

```scala
for {
  cli <- ??? // conjure up a RPC client somehow
  address <- cli.getNewAddress
  _ <- cli.generateToAddress(100, address)
} yield cli
```

Or more compactly: 

```scala
cli.getNewAddress.flatMap(cli.generateToAddress(100, _))
```

**Comment by cwaldron97** [comment]
Solved by #728 and #712 which creates node pairs and using `generatetoaddress` instead of `generate`. These are also implemented in #550 if you are looking for examples on how to use them
